extensions/api/auth-basic/build.gradle.kts

```diff
@@ -25,6 +25,14 @@ dependencies {
     api(project(":extensions:api:auth-spi"))
     api(project(":spi:core-spi"))
     implementation("jakarta.ws.rs:jakarta.ws.rs-api:${rsApi}")
+    implementation("org.junit.jupiter:junit-jupiter:5.8.1")
+    testImplementation("org.assertj:assertj-core:3.21.0")
+    testImplementation("org.mockito:mockito-inline:3.6.28")
```

Comment:
not needed (mockito-core should be added by default, and we don't use any feature from the `inline` package)